### PR TITLE
chore: Fix changelog formatting (PROJQUAY-3835)

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,603 +1,587 @@
...
+- [d9dbbd88](https://github.com/quay/quay/commit/d9dbbd88dbb9a1da95ef46195bcfb5458db6a7b8): basic support for cosign in the UI (PROJQUAY-3965) ([#1380](https://github.com/quay/quay/issues/1380))
+### V2auth
 - [fd9a6b2e](https://github.com/quay/quay/commit/fd9a6b2e606db6d20bb079f4c3a942e176f83a0c): Check for user before creating org (PROJQUAY-3766) ([#1315](https://github.com/quay/quay/issues/1315))
- -  [#1315](https://github.com/quay/quay/issues/1315)### Note
-
-that we're using ubi8/node-10, which is deprecated - but this is
-the version previously installed (before the nodejs package disappeared
-on us).
-
-### NOTE
```
Comment (dmage) [inline CHANGELOG.md:65]: This is from https://github.com/quay/quay/commit/ac6a8d2f8b93bcff032d3b2dfc57e07432a96709

I don't think we need `NOTE` from our commits in the change log, so removed them in this commit. We still can use `BREAKING CHANGE`.

cc @flavianmissi

Comment (flavianmissi) [reply CHANGELOG.md:65]: I didn't even know this ended up formatted like that here, thx for the ping @dmage.
And this is definitelly something not worth mention in the release notes, with pull-thru currently being tech-preview.

**Comment by HammerMeetNail** [comment]
We have a Jira floating around somewhere to track this, I'll see if I can dig it up and update the title. Long term I'd really like to see us drop the changelog completely from the repo and just provide reference to the release docs.

**Comment by HammerMeetNail** [comment]
I think this is the Jira https://issues.redhat.com/browse/PROJQUAY-3835

**Review by HammerMeetNail — APPROVED** [review]
LGTM

**Pull request merged**